src/Traversal/Sdk/Sdk.targets

```diff
@@ -236,10 +238,23 @@
              ContinueOnError="$([MSBuild]::ValueOrDefault('$(PublishContinueOnError)', '$(ContinueOnError)'))" />
   </Target>
   
-  <!--
-    Traversal projects do not build an assembly so dependent projects shouldn't get a path to the target.  Override the GetTargetPath to do nothing.
-  -->
-  <Target Name="GetTargetPath" />
+  <Target Name="GetTargetPath"
```

Comment:
@ViktorHofer I think `GetTargetPath` isn't available for a multi targeting build. So if one of the project references is multing-targeting (e.g, `<TargetFrameworks>net9.0;net10</TargetFrameworks>`), this would fail because the project doesn't have the target defined. What should be the right behavior here?